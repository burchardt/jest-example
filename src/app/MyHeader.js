import React from "react";

export default class MyHeader extends React.Component {
    render() {
        return(
            <h1>{this.props.header}</h1>
        );
    }
}