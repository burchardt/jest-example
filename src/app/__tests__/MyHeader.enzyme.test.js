import React from 'react';
import { shallow, mount} from 'enzyme';

import MyHeader from '../MyHeader';

describe('<MyHeader />', () => {
    it('should render a snapshot with a right header', () => {
        const wrapper = shallow(
            <MyHeader header="Header!!!" />
        );
        // Generate and match a shallow snapshot
        expect(wrapper).toMatchSnapshot();
    });
});