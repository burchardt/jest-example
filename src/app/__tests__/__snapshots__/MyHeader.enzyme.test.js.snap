// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyHeader /> should render a snapshot with a right header 1`] = `
<h1>
  Header!!!
</h1>
`;
